__tests__/reviews.test.js

```diff
@@ -206,4 +206,106 @@ describe("GET /api/reviews", () =>{
             })
         })
     })
+
+    test("status 200: results sorted by title descending", () =>{
+        return request(app)
+        .get("/api/reviews?sort_by=title")
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.reviews).toBeSortedBy("title", {descending : true})
+        })
+    })
+
+    test("status 200: results sorted by category descending", () =>{
+        return request(app)
+        .get("/api/reviews?sort_by=category")
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.reviews).toBeSortedBy("category", {descending : true})
+        })
+    })
+
+    test("status 200: results sorted by designer descending", () =>{
+        return request(app)
+        .get("/api/reviews?sort_by=designer")
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.reviews).toBeSortedBy("designer", {descending : true})
+        })
+    })
+
+    test("status 200: results sorted by owner descending", () =>{
+        return request(app)
+        .get("/api/reviews?sort_by=owner")
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.reviews).toBeSortedBy("owner", {descending : true})
+        })
+    })
+
+    test("status 200: results sorted by votes descending", () =>{
+        return request(app)
+        .get("/api/reviews?sort_by=votes")
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.reviews).toBeSortedBy("votes", {descending : true})
+        })
+    })
+
+    test("status 200: results sorted by comment_count descending", () =>{
+        return request(app)
+        .get("/api/reviews?sort_by=comment_count")
+        .expect(200)
+        .then(({body}) =>{
+            expect(body.reviews).toBeSortedBy("comment_count", {descending : true})
+        })
+    })
+
+    test("status 200: results arrive in ascending order", () =>{
```

Comment:
Could do with a little clarification in the descriptions about the difference between this test and the one above/ make it explicit that desc is the default but asc would need to be passed in.